lib/src/clock-in-out/clock-in/clock_in_middleware.dart

```diff
@@ -0,0 +1,45 @@
+part of 'clock_in.dart';
+
+class ClockInMiddleware extends AuthenticationMiddleware {
+  const ClockInMiddleware();
+
+  @override
+  Future<dynamic> call(HttpRequest req, HttpResponse res) async {
```

Comment:
@NazarenoCavazzon we should check if the user belongs, in any way, to the organization before letting him clock in